src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/DataElementsCard.tsx

```diff
@@ -21,6 +21,26 @@ const DataElementsCard = (props: {
     selectedDataElement,
     setSelectedDataElement,
   } = props;
+  let negationRationale = false;
+  // https://ecqi.healthit.gov/mcw/2020/qdm-attribute/negationrationale.html  (list of all categories that use negation rationale)
+  // we want to display negation rationale only in instances where the cql uses a qdm status that references an action.
+  if (selectedDataElement && selectedDataElement?.qdmStatus) {
+    const status = selectedDataElement.qdmStatus;
+    if (status) {
+      const negationRationaleActions = {
+        order: true,
+        recommended: true,
+        performed: true,
+        communication: true,
+        dispensed: true,
+        applied: true,
+      };
+      if (negationRationaleActions[status] === true) {
+        negationRationale = true;
+      }
+    }
+  }
```

Comment:
this might not be applicable to all the cases. e.g. EncounterPerformed doesn’t have negation but with the above logic, it does.
I believe all the data elements that can be negated have the `negationRationale` attribute. How about this?
`const negationRationale = selectedDataElement.hasOwnProperty("negationRationale")`
```suggestion
  let negationRationale = false;
  // https://ecqi.healthit.gov/mcw/2020/qdm-attribute/negationrationale.html  (list of all categories that use negation rationale)
  // we want to display negation rationale only in instances where the cql uses a qdm status that references an action.
  if (selectedDataElement && selectedDataElement?.qdmStatus) {
    const status = selectedDataElement.qdmStatus;
    if (status) {
      const negationRationaleActions = {
        order: true,
        recommended: true,
        performed: true,
        communication: true,
        dispensed: true,
        applied: true,
      };
      if (negationRationaleActions[status] === true) {
        negationRationale = true;
      }
    }
  }
```